apps/mobile/src/components/sheets/sheet.layout.tsx

```diff
@@ -1,22 +1,21 @@
-import { ReactNode, RefObject } from 'react';
+import { ReactElement, RefObject } from 'react';
```

Comment:
I haven't seen `ReactElement` before, nice! Is that something you recommend we use more? 

Like a better `JSX.Element`?

Reply:
Depends on the use case. ReactElement is a subset of ReactNode–basically just JSX, whereas ReactNode can be a literal string, number, null, etc. Icon doesn't expect literal values, so it made sense to narrow it down here.